capstone/client/src/components/ListPage.js

```diff
@@ -313,9 +315,16 @@ class ListPage extends Component {
         </ButtonGroup>
         <Grid container alignItems="stretch">
           <Grid id="distance-list-container" item component={Card} xs>
-            <p>I would like to choose from stores in a</p>
+            <p>Select a distance preference</p>
             <RadioGroup id="distance-list" value={this.state.distanceValue}>
-              {distances} 
+              {distances}
+              <FormControlLabel
+                control={<Radio name={"None"}/>}
+                label={"None"}
+                value={MAX_JAVA_INTEGER}
```

Comment:
Why do we set value to this number? Is it ok to just leave it empty?

Reply:
Leaving it empty would make it so all the stores would be filtered out since 0 is less than any distance value. I can add a check in the backend that doesn't filter if this value is 0 if that's better.

Reply:
Yeah I don't think 0 is a valid param so you should handle that in the API. I'll approve this and feel free to fix in a following change.